src/affinder/load_tf.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+import numpy as np
+from magicgui import magic_factory
+from napari.layers import Layer
+
+
+@magic_factory(call_button='Load', affine={'mode': 'r'})
+def load_affine(layer: Layer, affine: Path):
```

Comment:
```suggestion


@magic_factory(call_button='Load', affine={'mode': 'r'})
def load_affine(layer: 'napari.layers.Layer', affine: Path):
```

Prevent the need to import napari, same as we did [here](https://github.com/jni/affinder/blob/40370a1ba81fa22db598906fa5e3fd4f40a60f3f/src/affinder/apply_tf.py#L38)